Adds colorized-man-pages module

seagoj opened the pull request:
Sets correct variables for less to colorize man pages. Basically implements oh-my-zsh's [colored-man-pages](https://github.com/robbyrussell/oh-my-zsh).

Eriner commented:
I don't think creating a new module for this functionality is worthwhile, especially since there is no associated README.md file with the module. I would suggest moving this into another module. Additionally, there's no reason to call subshells to `printf`, as that is an expensive operation and it can be done without it. See the Arch wiki about colorized man pages for examples.

If you would like to make those changes I'd be happy to PR this.

seagoj commented:
I've added a README, moved the man function to the functions directory and implemented it using the method described in Arch wiki.

I don't have an issue combining this with another module, but I'm not sure which one it would make sense to merge it with. Possibly syntax-highlighting?

Eriner commented:
This functionality was added in commit 7a289c4.
